### ISSUE TITLE
Issue with Security module

### ISSUE THREAD

**Issue by cameronbourgeois** [Firesphere/silverstripe-csp-headers#20]
When I enable nonce with `useNonce: true` in my yml config, I get the following error when I try to log into the admin section:

```
[Emergency] Uncaught BadMethodCallException: Object->__call(): the method 'getNonce' does not exist on 'SilverStripe\Security\Security'
```

This is without me having added the nonce into any templates with `$Nonce` yet.

Everything works fine otherwise. The nonce is included in my CSP policy and gets added correctly if I do use `$Nonce` in my templates.

Using the following versions of the relevant modules:
```
firesphere/cspheaders: 1.0.1
silverstripe/framework: 4.5.2
```

Any ideas?

**Comment by Firesphere** [comment]
Yes, please see issue #15

**Comment by Firesphere** [comment]
Resolved in version 1.0.3
